code/ui/manager/src/components/layout/Layout.tsx

```diff
@@ -130,51 +136,70 @@ export const Layout = ({ managerLayoutState, setManagerLayoutState, ...slots }:
       panelPosition={managerLayoutState.panelPosition}
       isDragging={isDragging}
       viewMode={managerLayoutState.viewMode}
+      breakpoint={breakpointMin600}
     >
       {showPages && <PagesContainer>{slots.slotPages}</PagesContainer>}
-      <ContentContainer>{slots.slotMain}</ContentContainer>
-      <SidebarContainer>
-        <Drag ref={sidebarResizerRef} />
-        {slots.slotSidebar}
-      </SidebarContainer>
-      {showPanel && (
-        <PanelContainer position={panelPosition}>
-          <Drag
-            orientation={panelPosition === 'bottom' ? 'horizontal' : 'vertical'}
-            position={panelPosition === 'bottom' ? 'left' : 'right'}
-            ref={panelResizerRef}
-          />
-          {slots.slotPanel}
-        </PanelContainer>
+      <ContentContainer breakpoint={breakpointMin600}>{slots.slotMain}</ContentContainer>
+      {isDesktop && (
+        <>
+          <SidebarContainer>
+            <Drag ref={sidebarResizerRef} />
+            {slots.slotSidebar}
+          </SidebarContainer>
+          {showPanel && (
+            <PanelContainer position={panelPosition}>
+              <Drag
+                orientation={panelPosition === 'bottom' ? 'horizontal' : 'vertical'}
+                position={panelPosition === 'bottom' ? 'left' : 'right'}
+                ref={panelResizerRef}
+              />
+              {slots.slotPanel}
+            </PanelContainer>
+          )}
+        </>
       )}
+      {isMobile && <MobileNavigation />}
     </LayoutContainer>
   );
 };
 
-const LayoutContainer = styled.div<LayoutState>(
-  ({ navSize, rightPanelWidth, bottomPanelHeight, viewMode, panelPosition, isDragging }) => {
+const LayoutContainer = styled.div<LayoutState & { breakpoint: string }>(
+  ({
+    navSize,
+    rightPanelWidth,
+    bottomPanelHeight,
+    viewMode,
+    panelPosition,
+    isDragging,
+    breakpoint,
```

Comment:
1. `isDragging` is unused now
2. I think `breakpoint` is a bit too non-descriptive, it could be anything.

Here's my suggestion

```suggestion
    mobileBreakpoint,
```